Makefile

```diff
@@ -295,6 +295,15 @@ acncli-image: ## build cni-manager container image.
 		IMAGE=$(ACNCLI_IMAGE) \
 		TAG=$(ACNCLI_PLATFORM_TAG)
 
+acncli-image-windows: ## build the cni image for windows
```

Comment:
probably better to test using dropgz in windows flavor, as that's what we will use in production

Reply:
I did just peek at `dropgz`, Paul and I don't know quite how it works, could we meet to talk about how a windows flavor would work?

Reply:
We're actually trying to move conflist responsibilities to CNS - CNS is the component which knows when the Network is actually ready, and this has implications for NodeReady and Pod SLAs in AKS since the presence of the conflist is used to indicate NodeNetworkReady

Reply:
That's neat, okay, so we'd need to make one more generator in cns's conflist generator, and set the config flag to true, awsome!